### ISSUE TITLE
About NTU60 and NTU120 dataset

### ISSUE THREAD

**Issue by HaleYan** [open-mmlab/mmaction2#1981]
Thanks for your great work !
In the experiment, you have done the experiment of NTU60-XSub, NTU60-XView and NTU120-XSub, NTU120-XSet and presented the results in the paper, but only NTU60-XSub and NTU120-XSub are provided [mmaction2/tools/data/skeleton/README](https://github.com/open-mmlab/mmaction2/tree/master/tools/data/skeleton), so I want to know How the results of NTU60-XView and NTU120-XSet are obtained. And in [mmaction2/tools/data/skeleton/README](https://github.com/open-mmlab/mmaction2/tree/master/tools/data/skeleton), how did you divide the training set and test set of the hmdb51 and ucf101 datasets. thanks

**Comment by hukkai** [comment]
@HaleYan  The results of NTU60-XView and NTU120-XSet might be supported later, but not guaranteed. You can email the author to see if they have the plan to release it. For UCF101 and HMDB51, we use the default training and validation set.

**Comment by hukkai** [comment]
It it the common practice to use `train1` unless otherwise stated.
https://github.com/open-mmlab/mmaction2/blob/41dca8f3825dc8d6876e6589be708581359406f1/configs/skeleton/posec3d/slowonly_kinetics400_pretrained_r50_u48_120e_ucf101_split1_keypoint.py#L95

**Comment by hukkai** [comment]
@HaleYan  If you have any further questions, feel free to re-open the issue.
